packages/dd-trace/src/opentracing/span.js

```diff
@@ -150,7 +150,8 @@ class DatadogSpan {
         parentId: parent._spanId,
         sampling: parent._sampling,
         baggageItems: Object.assign({}, parent._baggageItems),
-        trace: parent._trace
+        trace: parent._trace,
+        localRoot: parent._localRoot
```

Comment:
This could refer to a span that can no longer be altered, and possibly cause memory leaks.